Maquetado/scss/tech.scss

```diff
@@ -0,0 +1,59 @@
+
+$tech-logo-size: 100px;
+$green: #BED23C;
+$text-big: 24px;
+$text-medium: 16px;
+$text-small: 12px;
+
+.row {
+    display: flex;
+    flex-flow: row nowrap; 
+    height: 100%;
+}
+
+.col {
```

Comment:
we can do this using a `::before` instead this should have the same size as the technology but translated

Reply:
@SKOLZ  According to your correction, I added the pseudo-element to the tech.css file and tried to debug the code.

Stay tuned n_n !!!

Reply:
you should rename this to `.tech` and apply the background here instead of adding it in the before